src/app/perfil-prestador/perfil-prestador.component.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { PerfilPrestadorComponent } from './perfil-prestador.component';
+import { ServicosComponent } from './servicos/servicos.component';
+import { FormularioServicoComponent } from './formulario-servico/formulario-servico.component';
+import { InformacoesPessoaisComponent } from '../perfil-usuario/informacoes-pessoais/informacoes-pessoais.component';
```

Comment:
Codacy found an issue: [`../perfil-usuario/informacoes-pessoais/informacoes-pessoais.component` import should occur after import of `ngx-mask`](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6332072)